lib/transit_tracker/transit/transit.ex

```diff
@@ -208,6 +208,13 @@ defmodule TransitTracker.Transit do
     Repo.all(query)
   end
 
+  def get_stop_by_children(id) do
+    query = from s in Stop,
+            where: ^id in s.children,
+            select: s
+    Repo.all(query)
```

Comment:
should this be `Repo.one(query)`? i don't think there's a possible case where this returns more than one stop